provisioning/service/src/Twin/TwinCollection.cs

```diff
@@ -174,9 +174,9 @@ public TwinMetadata GetMetadata()
         /// <returns>DateTime instance representing the LastUpdated time for this property.</returns>
         /// <exception cref="NullReferenceException">Thrown when the TwinCollection metadata is null.
         /// An example would be when the twin collection class is created with the default constructor.</exception>
-        public DateTime GetLastUpdatedOnUtc()
+        public DateTimeOffset GetLastUpdatedOnUtc()
         {
-            return (DateTime)_metadata[LastUpdatedName];
+            return (DateTimeOffset)_metadata[LastUpdatedName];
```

Comment:
We'll have a merge conflict on these twin changes, but no worries. It'll be easy to resolve.

Reply:
Because I already updated it to be DateTimeOffset.